### PR TITLE
feat: add badge component, modify icon element

### PR DESCRIPTION
**Pull request by ritik-patel05**
Modification available for badge

- Position
  - top right 
  - top left
  - bottom right
  - bottom left
- Color
  - Primary
  - Secondary
- Size
  - small
  - default
  - medium
  - large

https://user-images.githubusercontent.com/67374176/152233754-36b27c53-597e-4589-8efe-2dd5600c796f.mp4

### CODE REVIEW

File: css/components/badge.css [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+}
+
+.badge .is-secondary {
+    background-color: var(--secondary);
+    color: var(--white);
+}
+
+/* ---- Position ---- */
+
+/* NOTE: No idea how the below code works */
```
Comment (FarhanMobashir) [inline css/components/badge.css:42]: you can use css grid for achiveing that top , left , right positioning by the idea of overlapping

**Review by FarhanMobashir — COMMENTED** [review]
you can use css grid for positioning as it has overlapping thing

**Review by shivampandey0 — COMMENTED** [review]
LGTM

File: css/components/badge.css (hunk 1 above)
Comment (ritik-patel05) [reply css/components/badge.css:42]: Will consider doing that in future.

**Pull request merged**